integration-cli/docker_cli_run_test.go

```diff
@@ -404,8 +446,15 @@ func (s *DockerSuite) TestRunVolumesFromSymlinkPath(c *check.C) {
 }
 
 func (s *DockerSuite) TestRunExitCode(c *check.C) {
-	testRequires(c, DaemonIsLinux)
-	_, exit, err := dockerCmdWithError("run", "busybox", "/bin/sh", "-c", "exit 72")
+	var (
+		exit int
+		err  error
+	)
+	if daemonPlatform == "windows" {
+		_, exit, err = dockerCmdWithError("run", WindowsBaseImage, "cmd", "/s", "/c", "exit 72")
```

Comment:
@jhowardmsft i think he was just referring to your "workaround" comment: https://github.com/docker/docker/pull/16312/files#r39983870

From that, it seems like once the new hire debugged the issue, we could update this piece of code, hence the TODO.